src/components/tables/TitleWithTags.tsx

```diff
@@ -1,15 +1,30 @@
+import { Box, Flex, Td, Text } from "@chakra-ui/react";
+import Link from "next/link";
+
 import {
   convertStringToArray,
   tagColors,
   transcriptsCategories,
 } from "@/utils";
-import { Box, Flex, Td, Text } from "@chakra-ui/react";
+import config from "@/config/config.json";
+import { resolveGHBranchUrl } from "@/utils/github";
+
+
+const resolveTranscriptUrl = (transcriptUrl: string | null) => {
+  if (transcriptUrl) {
+    const { path, dir } = resolveGHBranchUrl(transcriptUrl)
```

Comment:
@kouloumos, there are some dead code in that file from when we were still going back and forth on the URL format. `resolveGHBranchUrl` is one of them.

the function to resolve all githuburl in the db i.e `transcriptUrl` or `branchUrl` is `resolveGHApiUrl`
https://github.com/bitcointranscripts/transcription-review-front-end/blob/d6aeacab216c3371de7170cc7c919942899ee605/src/utils/github.ts#L248